packages/web/src/components/welcome-modal/WelcomeModal.tsx

```diff
@@ -0,0 +1,107 @@
+import { useCallback } from 'react'
+
+import { fillString } from '@audius/common'
+import {
+  Button,
+  ButtonType,
+  Flex,
+  IconArrowRight,
+  Text,
+  IconCloudUpload,
+  Avatar
+} from '@audius/harmony'
+import { Modal } from '@audius/stems'
+import { push as pushRoute } from 'connected-react-router'
+import { useDispatch, useSelector } from 'react-redux'
+
+import { useModalState } from 'common/hooks/useModalState'
+import {
+  getNameField,
+  getProfileImageField
+} from 'common/store/pages/signon/selectors'
+import Drawer from 'components/drawer/Drawer'
+import { isMobile as getIsMobile } from 'utils/clientUtil'
+import { TRENDING_PAGE, UPLOAD_PAGE } from 'utils/route'
+
+const messages = {
+  welcome: 'Welcome to Audius%0! 🎉',
+  startListening: 'Start Listening',
+  upload: 'Upload'
+}
+
+const isMobile = getIsMobile()
+const Container = isMobile ? Drawer : Modal
+
+export const WelcomeModal = () => {
+  const dispatch = useDispatch()
+  const { value: userName } = useSelector(getNameField)
+  const profileImage = useSelector(getProfileImageField)
+  const [isOpen, setIsOpen] = useModalState('Welcome')
+
+  const onClose = useCallback(() => {
+    setIsOpen(false)
+  }, [setIsOpen])
+
+  const handleClickListen = useCallback(() => {
+    dispatch(pushRoute(TRENDING_PAGE))
+    onClose()
+  }, [dispatch, onClose])
+
+  const handleClickUpload = useCallback(() => {
+    dispatch(pushRoute(UPLOAD_PAGE))
+    onClose()
+  }, [dispatch, onClose])
+
+  return (
+    <Container isOpen={isOpen} onClose={onClose} size='small'>
+      <Flex
+        h='96px'
+        justifyContent='center'
+        css={{
+          zIndex: 1,
+          backgroundColor: '#ddd',
+          background: `url("${profileImage?.url}")`,
+          ...(!isMobile && {
```

Comment:
check out what we did on select-genre-ui, i think we should at least be consistent with how we do this blur. mabye we just create a helper component for all the pages?

Reply:
these two seem p different. I agree that the pages should use the same component, but this one is sized and spaced a bit differently. I'll try to make the blur consistent though